src/main/cpp/RLBotInterface/RLBotMessages/PacketStructs/RigidBodyStructs.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef RAWPHYSICSSTRUCTS_HPP
+#define RAWPHYSICSSTRUCTS_HPP
+
+#include "LiveDataPacket.hpp"
+#include "PacketStructs.hpp"
+
+struct Quaternion
+{
+	float X;
+	float Y;
+	float Z;
+	float W;
+};
+
+struct RigidBodyState
+{
+	PyStruct::Vector3 		Location;
+	Quaternion 				Rotation;
+	PyStruct::Vector3		Velocity;
+	PyStruct::Vector3		AngularVelocity;
+	unsigned long			Frame;
```

Comment:
I'd have put Frame first so it matches the other definition, but it's not a huge deal

Reply:
Also, to match the RL definition, it should just be `int`.

Reply:
Good point, will fix both! I guess I'll watch and see what happens when it rolls over...